spec/models/accounting/quickbooks/transaction_creator_spec.rb

```diff
@@ -2,28 +2,89 @@
 
```

Comment:
@X0nic I've made a guess about how best to merge the earlier develop (which includes changes from 6280) with this file. If you look over develop once the release goes out and something isn't right, just let me know.